### DISCUSSION TITLE
Insert variable value inside text event

### DISCUSSION THREAD

**Discussion by franpgn** [dialogic-godot/dialogic#1743] [Q&A]
I'm currently trying to insert a variable value inside of a text event to implement a score system. How can I do it? Since I'm new to Dialogic, I don't know if there is a better way to do it.

**Comment by Jowan-Spooner** [comment]
Hello @franpgn,
I'll assume you are using dialogic 2.x.

This can be done like this:
`{MyVariable}`
You can find more information on this in the variables editor at the right side.
This also works for autoload variables:
`{MyAutoload.variable}`
Or even autoload methods
`{MyAutoload.get_score()}`
Hope this helps.
